readmes/README.macOS.md

```diff
@@ -1,153 +1,140 @@
+    (1 row)
+
+    postgres=# select * from gp_segment_configuration;
+     dbid | content | role | preferred_role | mode | status | port |          hostname           |           address           |                                                 datadir                                                  | warehouseid 
+    ------+---------+------+----------------+------+--------+------+-----------------------------+-----------------------------+----------------------------------------------------------------------------------------------------------+-------------
+        1 |      -1 | p    | p              | n    | u      | 8000 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/qddir/demoDataDir-1         |           0
+        8 |      -1 | m    | m              | s    | u      | 8001 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/standby                     |           0
+        3 |       1 | p    | p              | s    | u      | 8003 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast2/demoDataDir1        |           0
+        6 |       1 | m    | m              | s    | u      | 8006 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast_mirror2/demoDataDir1 |           0
+        2 |       0 | p    | p              | s    | u      | 8002 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast1/demoDataDir0        |           0
+        5 |       0 | m    | m              | s    | u      | 8005 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast_mirror1/demoDataDir0 |           0
+        4 |       2 | p    | p              | s    | u      | 8004 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast3/demoDataDir2        |           0
+        7 |       2 | m    | m              | s    | u      | 8007 | hashdatas-MacBook-Pro.local | hashdatas-MacBook-Pro.local | /Users/hashdata/Documents/GitHub/upstream/cloudberrydb/gpAux/gpdemo/datadirs/dbfast_mirror3/demoDataDir2 |           0
+    (8 rows)
+
+    postgres=# 
+    ```
+
+    > [!NOTE]
+    > If the following problem occurs when connecting to the master:
+    >
+    >     ```
+    >     psql: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed: No such file or directory
+	>     Is the server running locally and accepting connections on that socket?
+    >     ```
```

Comment:
is this formatting right? on [rendered file view](https://github.com/cloudberrydb/cloudberrydb/blob/84d2d6a33ba1d801c975b89ab608820c2536c1e0/readmes/README.macOS.md) it seems wrong.
<img width="277" alt="image" src="https://github.com/cloudberrydb/cloudberrydb/assets/7938763/571b883b-08df-4b2d-908e-2263bbc39ff4">

except from this, all technicalities LGTM.

Reply:
No. See https://github.com/cloudberrydb/cloudberrydb/pull/220#discussion_r1336532933

Reply:
I've corrected the indentation, so you can take a look